src/sentry/seer/endpoints/organization_seer_onboarding_check.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import logging
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.constants import ObjectStatus
+from sentry.integrations.services.integration import integration_service
+from sentry.integrations.types import IntegrationProviderSlug
+from sentry.models.options.project_option import ProjectOption
+from sentry.models.organization import Organization
+from sentry.models.repositorysettings import RepositorySettings
+from sentry.ratelimits.config import RateLimitConfig
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
+
+logger = logging.getLogger(__name__)
+
+
+def check_github_integration(organization_id: int) -> bool:
+    """Check if the organization has an active GitHub or GitHub Enterprise integration."""
```

Comment:
l: Should we name this something more generic since we'll be adding more support down the line?

```suggestion
def check_scm_integration(organization_id: int) -> bool:
    """Check if the organization has an active and supported SCM, e.g. GitHub or GitHub Enterprise integration."""
```